website/docs/getting-started/flashing-nrf52.md

```diff
@@ -8,12 +8,44 @@ import TabItem from '@theme/TabItem';
 
 ## Pre-requisits
 
-Please ensure that you use a data USB cable, as some cables provide power only and not the data lines.
+Please ensure that you use a data USB cable, as some cables provide power only and not the data lines. Many popular brands (ie: Anker) are also "power" only.
```

Comment:
Not really needed, inferred by previous comment.

Reply:
Done - cleaned up (overwrote latest push - see new file)